service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/CaseSubmissionController.java

```diff
@@ -76,9 +78,20 @@ public AboutToStartOrSubmitCallbackResponse handleAboutToStartEvent(
 
         return AboutToStartOrSubmitCallbackResponse.builder()
             .data(data)
+            .errors(validate(mapper.convertValue(data, CaseData.class)))
             .build();
     }
 
+    private List<String> validate(CaseData caseData) {
+        ImmutableList.Builder<String> builder = ImmutableList.builder();
+
+        if ("FPLA".equals(caseData.getCaseLocalAuthority())) {
```

Comment:
I would prefer this if statement the other way around, I think that makes more logical sense.

Reply:
This is trick to avoid null checks. Obviously I am aware that `caseData.getCaseLocalAuthority` should be populated at the time.